doc/source/examples/dem/3d-small-scale-rotating-drum/3d-small-scale-rotating-drum.rst

```diff
@@ -0,0 +1,201 @@
+        set gy            		 						= -9.81
+        set gz            		 						= 0.0
+        set number of particle types	                = 1
+            subsection particle type 0
+            set size distribution type					= uniform
+                set diameter            	 			= 0.003
+                set number              				= 20000
+                set density particles  	 				= 2500
+                set young modulus particles         	= 100000000
+                set poisson ratio particles          	= 0.24
+                set restitution coefficient particles	= 0.97
+                set friction coefficient particles      = 0.3
+                set rolling friction particles         	= 0.01
+        end
+        set young modulus wall            				= 100000000
+        set poisson ratio wall            				= 0.24
+        set restitution coefficient wall           		= 0.85
+        set friction coefficient wall         			= 0.35
+        set rolling friction wall         	      	  	= 0.01
+    end
+
+
+Model Parameters
+~~~~~~~~~~~~~~~~~
+
+In this example, we use the ``dynamic`` load balancing method. This method checks frequently if load balancing should be applied based on a user inputted frequency. Load balancing is dynamically applied if a certain condition is applied. More details regarding load balancing are explained in the `Rotating Drum example <../rotating-drum/rotating-drum.html>`_. 
+
+.. code-block:: text
+
+    subsection model parameters
+      set contact detection method 		   	 = dynamic
+      set dynamic contact search size coefficient	 = 0.8
+      set neighborhood threshold			 = 1.3
+      set load balance method				 = dynamic
+      set load balance threshold			 = 0.5
+      set dynamic load balance check frequency		 = 10000
+      set particle particle contact force method         = hertz_mindlin_limit_overlap
+      set particle wall contact force method             = nonlinear
+      set integration method				 = velocity_verlet
+    end
+
+Boundary Condition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The rotation of the cylinder is applied using a rotational boundary condition with a value of 1 rad/s over the x axis. Based on deal.II boundary colouring, the hull of the cylinder (rotating drum) has an id = 0.
+
+.. code-block:: text
+
+    subsection DEM boundary conditions
+      set number of boundary conditions         = 1
+        subsection boundary condition 0
+            set boundary id					= 0
+            set type              				= rotational
+            set rotational speed				= 1
+            set rotational vector x				= 1
+            set rotational vector y				= 0
+            set rotational vector z				= 0
+        end
+    end
+
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The end time of the simulation is 5 seconds in real time. We set the time equal to 5 seconds as the simulation is restarted after the packing dem simulation which was run for 2 seconds in real time.
```

Comment:
```suggestion
The end time of the simulation is set to 5 seconds in real-time. Since the simulation restarts after the 2 seconds of packing, the actual rotation of the drum will be simulated for 3 seconds.
```